app/views/RoomView/index.js

```diff
@@ -227,7 +227,7 @@ export default class RoomView extends React.Component {
 	componentWillUnmount() {
 		this.mounted = false;
 		const { editing, replying } = this.props;
-		if (!editing && this.messagebox && this.messagebox.current && this.messagebox.current.text) {
+		if (!editing && this.messagebox && this.messagebox.current) {
```

Comment:
are u sure this is needed `&& this.messagebox && this.messagebox.current`

Reply:
it is necessary. If `this.messagebox.current` is undefined then on next line `const { text } = this.messagebox.current;` an error will be thrown: 'cannot destructure value text of undefined'